model/trace/database.yaml

```diff
@@ -453,6 +454,28 @@ groups:
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
       - ref: server.address
       - ref: server.port
+      - id: cluster_name
+        type: string
+        requirement_level:
+          recommended: >
+            Should be collected from the relevant HTTP response header if available.
+        tag: call-level-tech-specific
+        brief: >
+          Represents the identifier of an Elasticsearch cluster.
+        examples: ["e9106fc68e3044f0b1475b04bf4ffd5f"]
+        note: >
+          For Elastic Cloud hosted deployents the "X-Found-Handling-Cluster" HTTP response header contains the cluster ID.
+      - id: cluster_instance
+        type: string
+        requirement_level:
+          recommended: >
+            Should be collected from the relevant HTTP response header if available.
+        tag: call-level-tech-specific
+        brief: >
+          Represents the identifier of the node/instance to which the request was routed.
+        examples: ["instance-0000000001"]
+        note: >
+          For Elastic Cloud hosted deployents "X-Found-Handling-Instance" HTTP response header contains the instance identifier.
```

Comment:
Any recommendation for non-cloud hosted deployments? Is such attribute available only on Cloud hosted?

Reply:
This is only available on cloud and makes the most practical sense there as the language clients communicate through a reverse proxy, so the instance isn't known in advance. With self-managed, the host in the URL would be enough to determine the node to which a request has been sent.

Reply:
I see, thanks for the explanation!. Then I think it makes sense to also instruct those users under managed on how/which value should be populated in this field, if they want. Without it, it might not be obvious. If you think this attribute only makes sense on Cloud scenarios, then we should make that clear. E.g., Under recommended you can add, should be collected only when using Elastic Cloud..

Reply:
I've updated the recommendation to mention that this applies to cloud only (for now).